tests/test_survey_schema_translation.py

```diff
@@ -230,6 +254,16 @@ def test_get_catalog():
         ]
         in actual_items
     )
+    assert (
```

Comment:
Can we test `contents -> content` and `list` as well?

Reply:
Its just `contents` now. Ive removed `content` but Ive added a test for list